sysutils/helm/Portfile

```
@@ -19,28 +19,49 @@ set latestVersion       helm-3.8
 subport helm-3.8 {
     set baseVersion     3.8
     set patchNumber     1
-    revision            0
-    checksums           rmd160  d295fd8debbc4793b510490419e271275b3dfab8 \
+    revision            1
+
+    if {${os.arch} eq "arm"} {
```

If I recall correctly, I think the guidance is to prefer `${build_arch}`:

```suggestion
    if {${build_arch} eq "arm64"} {
```

---

Where's that guidance? Nobody updates the wiki anymore I've lost track on what's the latest.

Also, this port is just downloading a prebuilt binary, it's not building on the system. The Portfile doesn't even support building on the system.

---

`build_arch` is the explicit configuration option exposed in the MacPorts configuration files. MacPorts implicitly sets it, and it can also be set by the user: https://guide.macports.org/chunked/internals.configuration-files.html

It is set even if the port does not have an explicit build phase.  The port for `terraform` is also simply downloading a pre-built binary, and uses it in exactly this way: https://github.com/macports/macports-ports/blob/df134795909778139be6702f3fa5397084384639/sysutils/terraform/Portfile#L146

---

Sorry to nitpick, but `${build_arch}` has the same values as `supported_archs`.

Meaning you need to match for `arm64`, not `arm`.